yb-voyager/cmd/importData.go

```diff
@@ -532,32 +533,9 @@ func splitDataFiles(importTables []string, taskQueue chan *SplitFileImportTask)
 				if offsetEnd > largestOffsetSoFar {
 					largestOffsetSoFar = offsetEnd
 				}
-				addASplitTask("", t, filepath, splitNum, offsetStart, offsetEnd, true, taskQueue)
-			}
-		}
-		log.Infof("Collect files which were generated but processing did not start.")
-		// schedule import task for them
-		createdButNotStartedRegexStr := fmt.Sprintf(".+/%s\\.(\\d+)\\.(\\d+)\\.(\\d+)\\.(\\d+)\\.[C]$", t)
-		createdButNotStartedRegex := regexp.MustCompile(createdButNotStartedRegexStr)
-		// fmt.Printf("created but not started regex = %s\n", createdButNotStartedRegex.String())
-		for _, filepath := range matches {
-			submatches := createdButNotStartedRegex.FindAllStringSubmatch(filepath, -1)
-			for _, match := range submatches {
-				// This means a match. Submit the task with interrupted = true
-				splitNum, _ := strconv.ParseInt(match[1], 10, 64)
-				offsetEnd, _ := strconv.ParseInt(match[2], 10, 64)
-				numLines, _ := strconv.ParseInt(match[3], 10, 64)
-				offsetStart := offsetEnd - numLines
-				if splitNum == LAST_SPLIT_NUM {
-					fileFullySplit = true
-				}
-				if splitNum > largestCreatedSplitSoFar {
-					largestCreatedSplitSoFar = splitNum
+				if !doneSplitRegexp.MatchString(filepath) {
```

Comment:
Why not just check if the file-name ends with `.D`?

Reply:
Just wanted the check to be strict `*.D`, generated based on the table name.
Your approach should also be fine.